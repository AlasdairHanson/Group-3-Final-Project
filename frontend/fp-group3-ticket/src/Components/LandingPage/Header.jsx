import React from "react";
import { Container } from "react-bootstrap";
const Header = () => {
  return (
    <Container className="pt-5">
      <h1 className="text-center whiteText">Old Data</h1>
    </Container>
  );
};
export default Header;
