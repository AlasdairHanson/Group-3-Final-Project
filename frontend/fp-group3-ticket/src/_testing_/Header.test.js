import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import {create} from 'react-test-renderer';
import Header from '../Components/LandingPage/Header';


  describe(`Test of Header`, () => {

    test('Renders QA HelpDesk link', () => {
        render(<Header />);
        const linkElement = screen.getByText(/old data/i);
        expect(linkElement).toBeInTheDocument();
      });

    test(`Header should match the snapshot every time it is rendered`, () => {
      const instance = create(<Header/>).toJSON(); 
      expect(instance).toMatchSnapshot();
    });
  });

