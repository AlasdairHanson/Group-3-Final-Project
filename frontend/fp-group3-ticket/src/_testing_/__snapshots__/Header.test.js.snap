// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test of Header Header should match the snapshot every time it is rendered 1`] = `
<div
  className="pt-5 container"
>
  <h1
    className="text-center whiteText"
  >
    QA HelpDesk
  </h1>
</div>
`;
